gpytorch/kernels/kernel.py

```diff
@@ -352,6 +352,49 @@ def covar_dist(
 
         return res
 
+    def expand(self, *sizes: Union[torch.Size, Tuple[int, ...]]) -> Kernel:
```

Comment:
I know this is the abstract kernel and doesn't have non-batch shapes, but should we still call this `expand_batch` to be unambiguous?

Reply:
That's a good idea. I'll update the PR.